pkg/daemon/update_test.go

```diff
@@ -453,21 +451,3 @@ func checkIrreconcilableResults(t *testing.T, key string, reconcilableError erro
 		t.Errorf("Different %s values should not be reconcilable.", key)
 	}
 }
-
-func TestSkipReboot(t *testing.T) {
-	// skipReboot is only honored with onceFrom != ""
-	d := &Daemon{
-		mock:       true,
-		onceFrom:   "test",
-		skipReboot: true,
-	}
-	require.Nil(t, d.reboot("", 0, nil))
-
-	// skipReboot in normal cluster run is just a no-op and we reboot anyway
-	d = &Daemon{
-		mock:       true,
-		onceFrom:   "",
-		skipReboot: true,
-	}
-	require.NotNil(t, d.reboot("", 0, exec.Command("true")))
-}
```

Comment:
I'm still reluctant to remove a test however easy and straightforward it is...if someone messes up with the skipReboot `if` this was going to catch it

Reply:
We [talked about these unit tests before](https://github.com/openshift/machine-config-operator/pull/862#issuecomment-503264620) right?

Reply:
yup, no blocking, was just pointing out my initial reaction when a test gets deleted :D